brickstrap.sh

```diff
@@ -214,9 +238,10 @@ function brp_sanity_check_cli_options()
 
 function brp_validate_cli_options()
 {
-    brp_sanity_check_cli_options "$@" && \
-    brp_validate_project_name && \
+    brp_sanity_check_cli_options "$@"
+    brp_validate_project_name
     brp_validate_component_names
+    brp_validate_qemu || [ $? -eq 255 ] # no QEMU specified = 255
```

Comment:
That's at line 86 of `brickstrap-utils.sh` in: `brp_find_binfmt`.

It's only checked once we actually can do the detection of the binfmt, but that makes little difference because as I said it's valid for it not be there (we can recover through guessing from `ARCH` if it is sane).